### ISSUE TITLE
Conda wrong Bowtie version

### ISSUE THREAD

**Issue by Glfrey** [mhalushka/miRge3.0#64]
Hi again @arunhpatil,

I'm playing with a conda installation of Mirge3.0 and thought I best let you know that the current conda distribution contains the wrong Bowtie version. When you try to run it you get:

```
bowtie error!: incorrect version. Require - bowtie (1.2.1, 1.2.2, 1.2.3, 1.3.0 or 1.3.1)                                                                                     
Use argument -pbwt <name of the directory>  
```

A quick check showed me bowtie version 1.0.0 is the current version. 

Easily rectifiable by users, just thought to let you know.

**Comment by arunhpatil** [comment]
Hi @Glfrey, 

Thank you for brining this to our attention. The most recent version of bowtie is 1.3.1, however conda installs a more suitable version which matches the machines dependencies. This can be overcome by updating bowtie. Once again thank you for bringing this to our attention, I will try to implement a flexible solution in the upcoming release. 

Thank you,
Arun.

**Comment by arunhpatil** [comment]
Hi @Glfrey,

I have added a fix to include bowtie 1.0.0. 

Thank you,
Arun.
